tools/python_api/test/test_scan_pandas_pyarrow.py

```diff
@@ -632,3 +633,23 @@ def test_pyarrow_map_offset(conn_db_readonly: ConnDB) -> None:
         idx += 1
 
     assert idx == 48
+
+def test_pyarrow_decimal(conn_db_readwrite: ConnDB) -> None:
```

Comment:
Can u test the following cases

- scan from pandas decimal
- export decimal to python object, pandas
- in cpp side, create a table with decimal column. Insert decimal into it and scan out of the table.

Open the issue about supporting decimal in Java, NodeJS, Rust